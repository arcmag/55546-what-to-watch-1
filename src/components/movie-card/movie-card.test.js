import React from "react";
import renderer from "react-test-renderer";
import MovieCard from "./movie-card.jsx";

it(`Test MovieCard`, () => {
  const tree = renderer
    .create(<MovieCard title={0} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
