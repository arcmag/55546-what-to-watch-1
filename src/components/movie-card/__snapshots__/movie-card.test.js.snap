// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test MovieCard 1`] = `
<article
  className="small-movie-card catalog__movies-card"
>
  <button
    className="small-movie-card__play-btn"
    type="button"
  >
    Play
  </button>
  <div
    className="small-movie-card__image"
  >
    <img
      alt="Macbeth"
      height="175"
      src="img/macbeth.jpg"
      width="280"
    />
  </div>
  <h3
    className="small-movie-card__title"
  >
    <a
      className="small-movie-card__link"
      href="movie-page.html"
    >
      0
    </a>
  </h3>
</article>
`;
